packages/@aws-cdk/aws-lambda/package.json

```diff
@@ -183,7 +183,8 @@
   },
   "nozem": {
     "ostools": [
-      "docker"
+      "docker",
```

Comment:
@rix0rrr I needed this to use nozem. WDYT?

Reply:
You need a newer version of Nozem. A newer version will do this automatically.

Reply:
Reverted